.github/workflows/test-with-sqlite.yml

```diff
@@ -8,10 +8,10 @@ jobs:
       matrix:
         active_record: [6.1.4, 6.0.4, 5.2.6]
         # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
-        ruby: [2.6, 2.7, '3.0']
+        ruby: [2.6, 2.7, 3.0]
```

Comment:
Are you sure it is safe to remove quotes here?

You can see that `3.0` without quotes is interpreted as `3` in the [build](https://github.com/vlado/activerecord-cte/runs/4412214646?check_suite_focus=true). I think that [container:3](https://github.com/vlado/activerecord-cte/blob/746129740a9730428dfd4570121fe5a212cf668f/.github/workflows/test-with-postgresql.yml#L6) would install lates Ruby version 3 and not latest `3.0.x` version which could become problematic very soon (Ruby 3.1 is in preview).

Reply:
Ah... you are right
I haven't thought about that, let me recover the quotes 👍